### PR TITLE
Update ediscovery-decryption.md

### PR DESCRIPTION
**Pull request by v-padum**
Update for accuracy of behavior in the tool

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/ediscovery-decryption.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ This role is assigned by default to the eDiscovery Manager role group on the **P
 
 Any rights-protected (RMS-protected) email messages included in the results of a Content search will be decrypted when you export them. Additionally, any file that's encrypted with a [Microsoft encryption technology](encryption.md) and is attached to an email message that's included in the search results will be decrypted when it's exported. This decryption capability is enabled by default for members of the eDiscovery Manager role group. This is because the RMS Decrypt management role is assigned to this role group by default. Keep the following things in mind when exporting encrypted email messages and attachments:
   
-- As previously explained, to decrypt RMS-protected messages when you export them, you have to export the search results as individual messages. If you export search results to a PST file, RMS-protected messages remain encrypted.
+- As previously explained, if you enable decryption of RMS-protected messages when you export them, you have to export the search results as individual messages. If you try to export search results to a PST file, RMS-protected messages will be exported as loose emails instead.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ediscovery-decryption.md]:
```suggestion
- As previously explained, if you enable decryption of RMS-protected messages when you export them, you have to export the search results as individual messages. If you attempt to export search results to a PST file, RMS-protected messages will be exported as individual emails instead.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@v-padum Please implement the suggested changes. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@markjjo Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@markjjo Kindly merge this PR. Thanks

**Pull request merged**